### PR TITLE
Update package_version

### LINKED ISSUE #13594: Update package_version
## Reason
Update to latest package as default

## Design
modify the package_version file to instruct Civet to use said version:
51cabeeae3805351bef72baea532777fde328ac1

## Impact
Better GUI moose-environment bash profile support. See PR: https://github.com/idaholab/package_builder/pull/204

### PR DESCRIPTION
**Pull request by milljm**
Update package_version to use latest available moose-environment.

Closes #13594

### CODE REVIEW

**Comment by milljm** [comment]
I have more to add... based on user issues during MOOSE workshop.

**Comment by milljm** [comment]
PLEASE ALERT ME BEFORE CLICKING MERGE.

Before clicking merge on this PR, I must move the files into position on the mooseframework.org server.

File: modules/doc/content/getting_started/installation/mac_pre_req.md [hunk 1]
```diff
@@ -20,3 +20,16 @@
   - Mojave 10.14: [!package!name arch=osx10.14]
   - High Sierra 10.13: [!package!name arch=osx10.13]
   - Sierra 10.12: [!package!name arch=osx10.12]
+
+## Modify your Bash Profile
+
+During the installation of one of the above moose-environment packages, you have two opportunities to allow the installer to modify your bash profile. This will allow the moose-environment module system to be made available by default with every new terminal window opened.
+
+- The first being available after clicking 'Customize' during the install. While in this pane, check the option "MOOSE Environment", and then continue by clicking 'Install'.
```
Comment (cticenhour) [inline modules/doc/content/getting_started/installation/mac_pre_req.md]: `The first being available` --> `The first is available` ?

File: modules/doc/content/getting_started/installation/mac_pre_req.md [hunk 2]
```diff
@@ -20,3 +20,16 @@
...
+
+- The second, is after the installer has completed you will be presented with a pop-up window alerting you to either allow the installer to make this change, or to do nothing. If you choose 'Cancel' at this point, know that you are now responsible for altering your bash profile yourself. In order to make the module system available, you must instruct your bash profile(s) to source the following file:
```
Comment (cticenhour) [inline modules/doc/content/getting_started/installation/mac_pre_req.md]: Remove comma after `The second` and add `, where` after `completed`

**Review by cticenhour — CHANGES_REQUESTED** [review]
The bits about bash profile modifications look good! My comments were all grammatical. Thanks again for doing this!

**Review by permcody — APPROVED** [review]

**Pull request merged**